Source/SimCacheModule/Source/ViewModels/TrackerViewModel.cpp

```diff
@@ -93,22 +111,45 @@ void TrackerViewModel::UnregisterEventListeners()
 
 void TrackerViewModel::OnCacheFound( const CacheFoundEvent& Event )
 {
-	GetViewEventDispatcher().FireEvent( Event );
+	GetViewEventDispatcher().FireEvent( UICacheFoundEvent() );
+}
+
+// -----------------------------------------------------------------------------
+
+void TrackerViewModel::SendUITrackedCacheChangedEvent( const CacheDefinition* Cache ) const
```

Comment:
@StevenFrost do you have any thoughts about the naming of the `UITrackedCacheChangedEvent`?

We're sending it from both `TrackerViewModel::OnTrackerLoaded` (corresponding to the `UITrackerLoadedEvent` from the view event dispatcher, basically from the tracker InGamePanel) and `TrackerViewModel::OnTrackedCacheChangedEvent` (corresponding to the `TrackedCacheChangedEvent` from the internal event dispatcher, basically from the cache tracker subsystem).

To me, the name `UITrackedCacheChangedEvent` doesn't feel like it corresponds more strongly with the `TrackedCacheChangedEvent` since one of the main pathways for it being called is from the `UITrackerLoadedEvent`.

Do you think it would make more sense to be named something like `UITrackerDataUpdateEvent`, since its main purpose is to update all the data the UI tracker requires?

Reply:
Sure, I think that sounds sensible!